MyWeightTests/UITableViewExtensionsTests.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  UITableViewExtensionsTests.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import XCTest
+@testable import MyWeight
+
+class InvarianteTableViewCell: UITableViewCell { }
+class EmojiTableViewCell😎: UITableViewCell { }
+class a0_$WÅ´Ø: UITableViewCell { }
+
+class UITableViewExtensionsTests: XCTestCase {
+    private var tableView: UITableView!
+    
+    override func setUp() {
+        super.setUp()
+        tableView = UITableView()
+    }
+    
+    override func tearDown() {
+        tableView = nil
+        super.tearDown()
+    }
+    
+    func testDefaultReuseIdentifier() {
+        var cellClass = UITableViewCell.self
+        var cellDefaultReuseIdentifier = cellClass.defaultReuseIdentifier
+        var expectedDefaultReuseIdentifier = "UITableViewCell"
+        XCTAssertEqual(cellDefaultReuseIdentifier, expectedDefaultReuseIdentifier)
```

Comment:
I have no idea. I've never used Expecta/Nimble.
I actually was thinking about building the app "100% third party free" (it's an interesting idea, isn't it?), but it's your call. If you think Nimble is a good reason to add an external lib, go for it (and then I advocate towards using CocoaPods). And then we will start a discussion about including [RxSwift](https://github.com/ReactiveX/RxSwift).